docs/classes/pyfar.coordinates.rst

```diff
@@ -2,6 +2,37 @@ Coordinates
 -----------
 
 .. automodule:: pyfar.classes.coordinates
+
+**Classes:**
+
+.. autosummary::
+
+   pyfar.Coordinates
+
+**Functions:**
+
+.. autosummary::
+
+   pyfar.classes.coordinates.cart2cyl
+   pyfar.classes.coordinates.cart2sph
+   pyfar.classes.coordinates.cyl2cart
+   pyfar.deg2rad
```

Comment:
maybe change the order, first rad2deg/deg2rad then the others?

Reply:
I thought it might be nicer to strictly stay with the alphabetic order. Maybe we can generally discuss if we want to import those as top level functions...

Reply:
also fine for me. should we rename the references now? like mentioned in the comment of the review?